src-tauri/tauri.conf.json

```diff
@@ -27,15 +27,16 @@
     "category": "DeveloperTool",
     "shortDescription": "",
     "longDescription": "",
-    "createUpdaterArtifacts": false
+    "createUpdaterArtifacts": "v1Compatible"
   },
   "plugins": {
     "updater": {
-      "active": false,
-      "dialog": false,
+      "windows": {
+        "installMode": "passive"
```

Comment:
what does passive mean?

Reply:
There will be a small window with a progress bar. The update will be installed without requiring any user interaction. Generally recommended and the default mode.
https://tauri.app/plugin/updater/#installmode-on-windows